db/entities/picture/picture.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+from datetime import date
+from db.entities.tableEntity.tableEntity import TableEntity
+from db.queries import queries
+
+class Picture(TableEntity):
+    picture_id: uuid
+    picture: dict
+    index_id: uuid
+    owner_id: uuid
+    
+    nbSeed: int
+    zoom: float
+    
+    uploadDate: date
+    parent: bool
+    source: str
+    format: str
+    height: int
+    width: int
```

Comment:
You have a lot of variables here that are not initialized in the `__init__`, if you want to give flexibility to your user, you can add `**kwargs` arguments and have a special function to assign the value pass to the correct one variable

Reply:
We decided that we are not using the object structure in the end